roles/builder/tasks/main.yml

```diff
@@ -1,4 +1,10 @@
 ---
+- name: Set fact
+  when: "'-' in builder_compose_type"
+  ansible.builtin.set_fact:
+    a: "{{ builder_compose_type }}"
```

Comment:
What's the motivation behind setting this `a` variable? I would suggest we just use `builder_compose_type` everywhere. Thoughts?